pkgs/development/compilers/ghc/7.10.3-binary.nix

```diff
@@ -15,6 +16,16 @@ let
   libEnvVar = stdenv.lib.optionalString stdenv.hostPlatform.isDarwin "DY"
     + "LD_LIBRARY_PATH";
 
+  bootPackages = [
```

Comment:
I agree my existing `bootPkgs` is not a good name, but as these are libraries built *with* GHC, but not necessary used to build GHC (e.g. cross compiling), I think I'd like something else. I think GHC calls these `wiredinPackages` in some places. @bgamari @angerman?

Reply:
Wired in packages is a different thing: https://ghc.haskell.org/trac/ghc/wiki/Commentary/Libraries#CouplingtoGHC